solutions/classes/car-sale-customer.js

```diff
@@ -0,0 +1,47 @@
+let fs = require("fs");
+let data = fs.readFileSync(0, "utf-8");
+let idx = 0;
+data = data.split("\n");
+
+function readLine() {
+  idx++;
+  return data[idx - 1];
+}
+
+// -------- Do NOT edit anything above this line ----------
+
+// Your class should be named `CarSell`.
+// Your method should be named `getPromisingCustomers`
+// Your method should return the indices of customers who are willing to pay greater than or equal to 90% of the car value
+class CarSell {
+  constructor(customerProposals) {
+    this.customerProposals = customerProposals;
+  }
+  getPromisingCustomers() {
+    let promisingcustomers = [];
+    let count = 0;
+    let flag = false;
```

Comment:
@Rahulkug We don't need `count` and `flag`. If there are no promising cutomers, the array will be of `0` length, in which case you can push `-1`